examples/theme-ui/README.md

```diff
@@ -0,0 +1,49 @@
+# Example app with [theme-ui](https://theme-ui.com/)
+
+This example features how to use [theme-ui](https://theme-ui.com/) with Remix.
+
+## How this implementation works
+
+Since [theme-ui](https://theme-ui.com/) is derived from [emotion](https://emotion.sh/), this example shows how we can leverage Emotion's server-side-caching to enable [theme-ui](https://theme-ui.com/) server side rendering.
+
+*This implementation was based off [Saas-UI Remix guide](https://www.saas-ui.dev/docs/core/installation/remix-guide).*
+
+### Theme-UI Related files
+
+```
+- app/
+  - styles/
    - context.tsx
    - createEmotionCache.tsx
```

Comment:
<img width="520" alt="image" src="https://user-images.githubusercontent.com/1500684/177376678-408aeb55-bbbb-405d-84e7-b236c1350ff1.png">

Do these newlines show up funny to anyone else?

Reply:
Looks like it renders fine so I'll just merge. Thanks!

Reply:
Odd, I didn't see those. Thanks for approving :)